scripts/kind-e2e/globalnet/role-globalnet.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I tried this patch locally and seeing the following errors.

E1128 08:09:28.444999       1 reflector.go:125] k8s.io/client-go/informers/factory.go:132: Failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:submariner:submariner-globalnet" cannot list resource "pods" in API group "" at the cluster scope
E1128 08:09:28.446803       1 reflector.go:125] k8s.io/client-go/informers/factory.go:132: Failed to list *v1.Service: services is forbidden: User "system:serviceaccount:submariner:submariner-globalnet" cannot list resource "services" in API group "" at the cluster scope
I1128 08:09:29.445311       1 reflector.go:160] Listing and watching *v1.Pod from k8s.io/client-go/informers/factory.go:132
E1128 08:09:29.446430       1 reflector.go:125] k8s.io/client-go/informers/factory.go:132: Failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:submariner:submariner-globalnet" cannot list resource "pods" in API group "" at the cluster scope
I1128 08:09:29.446923       1 reflector.go:160] Listing and watching *v1.Service from k8s.io/client-go/informers/factory.go:132
E1128 08:09:29.449149       1 reflector.go:125] k8s.io/client-go/informers/factory.go:132: Failed to list *v1.Service: services is forbidden: User "system:serviceaccount:submariner:submariner-globalnet" cannot list resource "services" in API group "" at the cluster scope

It looks like this rbac is not configured.

Reply:
Aaah, should I be using deploytool=operator?

Reply:
working fine when using operator.